release-deployment.md

```diff
@@ -269,3 +273,12 @@ The process for tagging and merging is a bit different when deploying a bundle,
    Click `Publish release`.
 
    **NOTE: the release tag will be 1 commit ahead of the bundle commit**
+
+1. Merge `master` into `develop`.
```

Comment:
Since this is a duplicate of the process we wrote above, what do you think about DRYing this up by simply linking to the step above on how to do this?